build.sh

```diff
@@ -3,4 +3,4 @@ if [ ! -d "$BUILD_DIR" ]; then
 	mkdir "$BUILD_DIR"
 fi
 
-g++ -Wall -Werror -Wpedantic src/main.cpp src/fs.cpp src/CommandHandlerer.cpp src/cc.cpp -std=c++17 -o build/main.obj
+g++ -Wall -Werror -Wpedantic src/main.cpp src/fs.cpp src/CommandHandlerer.cpp src/cc.cpp -std=c++17 -Ilib -o build/main.obj
```

Comment:
now the advantage of using cmake is that, whatever your platform is, you can do:
```
cmake -DCMAKE_BUILD_TYPE=Release -S . -B build
cmake --build build --config Release
```
The `-S .` tells you the source folder, the `-B build` creates the build folder and builds in it (it is the equivalent of `mkdir build && cd build && cmake ..`).
You can specify if you don't want to use the default generator (Makefile on Linux, but Ninja - visual c++ 20xx or mingw on Windows...).